### ISSUE TITLE
Add Shelly GAS Addon Manipulator / Valve support

### ISSUE THREAD

**Issue by pizzulicchio** [enesbcs/shellyteacher4domo#53]
Good day!
I have got a shelly gas sensor with the shelly valve. The valve and its status (Open/Close) is not detected from your plugin.
Can you support this implementation ?

have a nice day

**Comment by enesbcs** [comment]
Yes, it can be done. According the shelly docs this is the valve mqtt path:
`shellies/shellygas-<deviceid>/valve/0/state	`

and this is the possible values, so it can be a selector type, not a binary:
```
unknown = Valve state is unknown
closed = Valve is closed
opened = Valve is opened
not_connected = Valve is not connected
failure = Valve failure detected
closing = Valve is in the process of closing
opening = Valve is in the process of opening
checking = Valve state is being checked
```

**Comment by pizzulicchio** [comment]
I can confirm all.
"closed" , "opened", and "not connected" are the more important states.
"closing" and "opening" are active during the changing position. 
if you need some logs from my mosquitto mqtt server or something else, let me know.

I'd like to use this integration for create an event, in domoticz,  that close the gas valve when you left home, and reopen when you came back. 
;-)
